junit-engine-api/src/main/java/org/junit/gen5/engine/ExecutionRequest.java

```diff
@@ -25,12 +25,16 @@
 
 	private final EngineExecutionListener engineExecutionListener;
 
+	private ConfigurationParameters configurationParameters;
+
 	private final Map<String, Object> attributes = new HashMap<>();
```

Comment:
When I coded around that line I was also like "let's remove them", but as we not sure about those parameters in our discussion, I didn't want to remove them right away.

Readying your comment now convinces me, that we should remove them now, and if we need them later, we could always add them again.

What about the others?